ci/helm/preview/values.yaml

```diff
@@ -43,7 +43,7 @@ nuxeo:
       pullPolicy: Always
     persistence:
       enabled: disable
-    packages: nuxeo-web-ui amazon-s3-online-storage /packages/nuxeo-coldstorage-package-*.zip
+    packages: nuxeo-web-ui-3.1.0-rc.59 amazon-s3-online-storage /packages/nuxeo-coldstorage-package-*.zip
```

Comment:
Could you explain why we are locking the web-ui version.

Reply:
we want to get the latest web-ui version - by default we get the last release candidate on connect pre-prod, but that doesn't have all the code we need for now.

Reply:
@andreacornaglia I think we don't have a version 3.1.0-rc.59 for WebUI.
maybe you want to use the version https://nos-preprod-connect.nuxeocloud.com/nuxeo/site/marketplace/package/nuxeo-web-ui?version=3.1.0-rc.059

Reply:
@doulba I want to use https://github.com/nuxeo/nuxeo-web-ui/tree/v3.1.0-rc.59

Reply:
Ah, I think the release didn’t happen.
the latest one is 3.1.0-rc.41

Reply:
yes, I've checked with Gabriel and indeed the version is.  https://nos-preprod-connect.nuxeocloud.com/nuxeo/site/marketplace/package/nuxeo-web-ui?version=3.1.0-rc.059 - updating